.gitignore

```diff
@@ -13,4 +13,17 @@ shared-*/
 !package.json
 
 # dynamically created by the build script
-conf-notes.txt
```

Comment:
This file should not be commited. Please remove from the commit

Reply:
I don't think I added the file, it was merely added to the .gitignore file (to prevent my IDE from always identifying it as something that needs to added).

Reply:
This commit has been removed from the PR.